cf_exporter.tf

```diff
@@ -10,9 +10,10 @@ resource "cloudfoundry_app" "cfpaasexporter" {
     username = var.docker_username
     password = var.docker_password
   }
+
+  command = "export USERNAME=${var.cf_functional_account.username};export PASSWORD=${var.cf_functional_account.password};paas-prometheus-exporter"
```

Comment:
This might capture the password in the /proc system. Anyone who has access to the CF space as developer can essentially get to the credentials, so this will not really improve things.

If there are still concerns then maybe it's better to isolate the paas exporter in its own space with limited accessiblity (e.g. only the CICD pipeline has access to this space for instance)

Reply:
I think this might actually cause leaking of the credentials since  the`command` is not a place where CF would expect sensitive data.

Reply:
CF uses ENVs for secrets by design e.g. binding services injects credentials in `VCAP_SERVICES`

More container native orchestrators like k8s and Nomad can do volume or bind mounts of secrets, which is IMO more elegant, but both options work fine